test-files/README.md

```diff
@@ -0,0 +1,36 @@
+# Transformer Unit Tests
+This folder holds the unit tests [TravisCI](https://travis-ci.org/) will run in it's build. 
+These are basic functional unit tests, checks for output formatting and typing. 
+All tests are written in and utilizing [pytest](https://docs.pytest.org/en/latest/). 
+In addition to this, [pylint](https://www.pylint.org/) will also be deployed. 
+Look into our organization's repository for our [pylint protocols](https://github.com/AgPipeline/Organization-info).
+
+### Running the tests
+Upon submitting a pull request Travis will build and run the testing modules automatically and will return a report with all passing or failing code. 
```

Comment:
'... with **a** passing or ...' - should 'all' be changed to 'a'?

Reply:
Changed the sentence to this, I think this probably looks better

"Upon submitting a pull request Travis will build and run the testing modules automatically and return a report with passing and failing code."

Reply:
I like it!